bag_unpacker/src/main/scala/uk/ac/wellcome/platform/archive/bagunpacker/services/Unpacker.scala

```diff
@@ -66,12 +66,16 @@ trait Unpacker {
     }
   }
 
-  private def buildMessageFor(srcLocation: ObjectLocation,
-                              error: UnpackerError): Option[String] =
+  protected def buildMessageFor(srcLocation: ObjectLocation,
+                                error: UnpackerError): Option[String] =
     error match {
       case UnpackerStorageError(_: DoesNotExistError) =>
         Some(s"There is no archive at $srcLocation")
 
+      case UnpackerUnarchiverError(_) =>
+        Some(
+          s"Error trying to unpack the archive at $srcLocation - is it the correct format?")
```

Comment:
`- not a valid .tar.gz`? or can we not necessarily tell and it only could be that? If so, then your suggestion is 👍

Reply:
We can’t really tell – in theory the unpacker supports any archive + compression type format.